packages/components/src/Dialog/Dialog.stories.tsx

```diff
@@ -66,16 +72,89 @@ export const Form: ComponentStory<typeof Dialog.Trigger> = args => {
   );
 };
 
-export const CustomTitleProps: ComponentStory<typeof Dialog.Trigger> = args => {
-  return (
-    <>
-      <Dialog.Trigger {...args}>
-        <Button variant="primary">Open</Button>
-        <Dialog closeButton aria-labelledby="my-cool-headline">
+export const CustomTitleProps: ComponentStory<typeof Dialog.Trigger> = args => (
+  <Dialog.Trigger {...args}>
+    <Button variant="primary">Open</Button>
+    <Dialog closeButton aria-labelledby="my-cool-headline">
+      <Headline id="my-cool-headline">This is a headline!</Headline>
+      <Text>This is some not so very long text.</Text>
+    </Dialog>
+  </Dialog.Trigger>
+);
+
+export const ScrollableContent: ComponentStory<
+  typeof Dialog.Trigger
+> = args => (
+  <Dialog.Trigger {...args}>
+    <Button variant="primary">Open</Button>
+    <Dialog closeButton aria-labelledby="my-cool-headline">
+      <Container>
+        <Header>
           <Headline id="my-cool-headline">This is a headline!</Headline>
-          <Text>This is some not so very long text.</Text>
-        </Dialog>
-      </Dialog.Trigger>
-    </>
-  );
-};
+        </Header>
+        <Body>
+          <Stack space="small">
+            <Text>This is some not so very long text.</Text>
+            <Box
+              pl="small"
+              pb="xsmall"
+              css={{ maxHeight: '200px', overflowY: 'auto' }}
+            >
+              <CheckboxGroup>
+                <Checkbox>One</Checkbox>
+                <Checkbox>Two</Checkbox>
+                <Checkbox>Three</Checkbox>
+                <Checkbox>Four</Checkbox>
+                <Checkbox>Five</Checkbox>
+                <Checkbox>Six</Checkbox>
+                <Checkbox>Seven</Checkbox>
+                <Checkbox>Eight</Checkbox>
+                <Checkbox>Nine</Checkbox>
+                <Checkbox>Ten</Checkbox>
+              </CheckboxGroup>
+            </Box>
+          </Stack>
+        </Body>
+        <Footer>
+          <Button variant="primary">Ok</Button>
+        </Footer>
+      </Container>
+    </Dialog>
+  </Dialog.Trigger>
+);
+
+export const StickyFooter: ComponentStory<typeof Dialog.Trigger> = args => (
```

Comment:
maybe we can add this in a demo too?

Reply:
in the docs you mean? I would like to wait for #2537 because this should make the Box obsolete